packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -561,6 +578,34 @@ class FileCollection extends DocumentCollection {
     }
   }
 
+  /**
+   *
+   * Updates the metadata attribute of a io.cozy.files
+   * Creates a new version of the file without having
+   * to upload again the file's content
```

Comment:
Why are we mentioning versioning here ? Is it different from `updateAttributes`. If not, should we mention versioning there too ?

Reply:
I think https://docs.cozy.io/en/cozy-stack/files/#patch-filesfile-id-and-patch-filesmetadata doesn't create a new version of the file. @nono do you confirm?

Reply:
You are right